server/mvcc/buckets/auth.go

```diff
@@ -29,20 +33,60 @@ var (
 	authDisabled = []byte{0}
 )
 
-func UnsafeCreateAuthBucket(tx backend.BatchTx) {
+type authBackend struct {
+	be backend.Backend
+	lg *zap.Logger
+}
+
+var _ auth.AuthBackend = (*authBackend)(nil)
+
+func NewAuthBackend(be backend.Backend, lg *zap.Logger) *authBackend {
```

Comment:
The parameter order doesn't seem consistent with others. Can we pass `lg` first?

Reply:
Done